package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "reservationsystem",
+  "version": "1.0.0",
+  "description": "A web app that enables the YDRC staff to coordinate the reservation process in their organization between all of them.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "client": "cd client && npm start",
+    "start": "nodemon start ./server/server.js && npm run client",
```

Comment:
You have to put ```nodemon``` inside ```dev``` script or anything else except ```start``` script. You can't  run the code on heroku with ```nodemon``` module

Reply:
done